docs/standard-library/http/reference/decorators.md

```diff
@@ -235,6 +235,36 @@ dec TypeSpec.Http.query(target: ModelProperty, queryNameOrOptions?: string | Typ
 | ------------------ | -------------------------------------------- | ------------------------------------------------------------------------------- |
 | queryNameOrOptions | `union string \| TypeSpec.Http.QueryOptions` | Optional name of the query when included in the url or query parameter options. |
 
+### `@route` {#@TypeSpec.Http.route}
+
+`@route` defines the relative route URI for the target operation
+
+The first argument should be a URI fragment that may contain one or more path parameter fields.
+If the namespace or interface that contains the operation is also marked with a `@route` decorator,
+it will be used as a prefix to the route URI of the operation.
+
+`@route` can only be applied to operations, namespaces, and interfaces.
+
+```typespec
+@route("/widgets")
+op getWidget(@path id: string): Widget;
+```
+
+```typespec
+dec TypeSpec.Http.route(target: Namespace | Interface | Operation, path: string, options?: object)
+```
+
+#### Target
+
+`union Namespace | Interface | Operation`
+
+#### Parameters
+
+| Name    | Type            | Description |
+| ------- | --------------- | ----------- |
+| path    | `scalar string` |             |
+| options | `model object`  |             |
```

Comment:
`@param` annotations needed in the docstrings to populate these.

Reply:
Forgot to add those, thanks